src/App.js

```diff
@@ -1,24 +1,16 @@
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
+import useFetch from "./hooks/useFetch";
 import Button from "./components/UI/Button/Button";
 import CharactersList from "./components/CharacterList";
 import "./App.css";
 
 function App() {
   console.log("App rendered");
-  const [apiDB, setApiDB] = useState();
+  const [apiDB, isLoading, error] = useFetch(
+    "https://thronesapi.com/api/v2/Characters"
```

Comment:
It is just a common practice (nothing technical), but it is more common to have constants on seperate file or at the top of the module.. `const URL = "https://thronesapi.com/api/v2/Characters"`